src/views/Challenge.vue

```diff
@@ -135,6 +139,9 @@ export default {
       }
       return false
     },
+    isFund7() {
+      return this.fund == 'f7'
```

Comment:
To make things more abstract, maybe it makes sense to change this condition to make a check for the existence of the properties on the first element of `this.proposals`. In this way we can avoid hard-coding fund names in the future.

Reply:
I've implemented a solution where we would be able to check each individual property so there's no problems even if the properties change in the following funds.